cypress/e2e/tests/rbac/custom-rules.test.ts

```diff
@@ -78,6 +81,17 @@ describe(["@tier2"], "Custom Rules RBAC operations", function () {
         });
     });
 
+    it("Verify triggered rule", function () {
```

Comment:
Can you remind me why we have these tests under rbac folder ? 

All admin tests can be under analysis folder ? I

Reply:
Because this suite covers custom rules analysis for admin, migrator and architect. I can move this single test to another file.

Reply:
For this test if you decide to leave it here. Please rename it to "Admin, Verify triggered rule".

For other custom rules that we wil just test with admin , let's create a file under analysis "custom_rules.test.ts